R/tcplhit2_core.R

```diff
@@ -79,6 +79,8 @@ tcplhit2_core <- function(params, conc, resp, cutoff, onesd,bmr_scale = 1.349, b
 
   # get error distribution
   errfun = params[["errfun"]]
+  if (is.null(errfun))
+    warning("'errfun' is missing in the output from tcplfit2_core. 'errfun' tracks the error distribution assumed for the model fits and should be provided in 'params' list.")
```

Comment:
Suggest just adding "-- see tcplfit2_core help for additional details." at the end of the error function.